.kitchen.yml

```diff
@@ -0,0 +1,52 @@
+---
+driver:
+  name: docker
+  hostname: git.ci.local
+  use_sudo: true
```

Comment:
Can we set this to false? Otherwise kitchen is not able to detect docker binary.